CODE_OF_CONDUCT.md

```diff
@@ -35,23 +35,23 @@ socioeconomic status, or other similar personal characteristics.
 Examples of behavior that contributes to creating a positive environment
 include:
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy toward other community members
+- Using welcoming and inclusive language
```

Comment:
Can we leave .md and .html files out of the prettier consideration? I tend to find its "opinions" on the former to be less than standards-based, and on the latter it can break LWC templates outright.

Reply:
I left it there b/c LBC team has it in their repo to ensure RFC MD files are in "proper" format. I've copied their MD linting config files as well. Are you sure we want to remove MD checks? I don't think I'm doing any HTML checking.